demo-projects/blog/index.js

```diff
@@ -24,16 +24,16 @@ const keystone = new Keystone({
   },
 });
 
-const authStrategy = keystone.createAuthStrategy({
-  type: PasswordAuthStrategy,
-  list: 'User',
-});
-
 keystone.createList('User', User);
 keystone.createList('Post', Post);
 keystone.createList('PostCategory', PostCategory);
 keystone.createList('Comment', Comment);
 
+const authStrategy = keystone.createAuthStrategy({
```

Comment:
For the sake of documenting the discussion If we are going to rely on a specific order of execution for these I feel like maybe rather than passing a string to createAuthStrategy perhaps we should pass the list itself?

```js
const list = keystone.createList('User', User);
const authStrategy = keystone.createAuthStrategy({
  type: PasswordAuthStrategy,
  list
});
```

Would you pass a promise or await createList?

Reply:
`createList` isn't async, it simply returns a `List` object, which could be passed in as an option to `createAuthStrategy` in a hypothetical future change to this API.

I would like to defer making such an API change to a seperate PR and discussion.